integr8ly/manifests
# bump keycloak-operator and integreatly-operator

## Motivation
This is required for creating a [Github](https://github.com/integr8ly/integreatly-operator/pull/61/files#diff-329c149b9583e9a4fcdb670a2ba866f0R316-R317) identity provider to be used for Launcher & Che in OpenShift 4.

## What
This adds the properties `linkOnly` to the [Keycloak Identity Provider](https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_identityproviderrepresentation) type.

## Why
The Github identity provider created for Launcher and Che in OpenShift 4 requires the `Account Linking Only` to be set to true.


## Verification Steps
1. Push the changes above to your app registry
 - Run `make push/all`
 - **Verify**: RHSSO (1.8.1) and Integreatly Operator (1.5.9) should be available in your app registry
2. Create an `OperatorSource` with your app registry
e.g.
```yaml
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  finalizers:
    - finalizer.operatorsources.operators.coreos.com
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators austincunningham
  endpoint: 'https://quay.io/cnr'
  publisher: austincunningham Publisher
  registryNamespace: austincunningham
  type: appregistry
```
 - **Verify**: RHSSO should become available in the Operator Hub
 - **Verify**: RHSSO CR should be available in the developer catalog
3. Install RHSSO to a specific namespace
 - **Verify**: An Install plan should be created
 - **Verify**: RHSSO operator should be deployed successfully

## Checklist:


## Additional Notes
Related PR: https://github.com/integr8ly/integreatly-operator/pull/61 and https://github.com/integr8ly/keycloak-operator/pull/90
